apps/frontend/src/libs/components/table-pagination/styles.module.css

```diff
@@ -0,0 +1,38 @@
+.container,
+.pagination-container,
+.rows-per-page,
+.change-page-buttons {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+}
+
+.container {
+	height: 32px;
+	font-size: 14px;
+	font-weight: 400;
+	line-height: 1.5;
+	color: var(--color-text-primary);
+
+	/* TODO: remove the folowing bg-color style after visual testing */
+	background-color: var(--color-background-secondary);
+}
+
+.pagination-container {
+	column-gap: 32px;
+}
+
+.rows-per-page {
+	column-gap: 8px;
+}
+
+.change-page-buttons {
+	column-gap: 7px;
+}
+
+.total-items-text,
+.rows-per-page-text,
+.page-text {
+	padding: 5.5px 0;
```

Comment:
round values and maybe instead we need to add padding to container? or make it display flex and just center everything